dhis-2/dhis-support/dhis-support-artemis/src/main/java/org/hisp/dhis/artemis/audit/legacy/DefaultAuditObjectFactory.java

```diff
@@ -36,9 +36,6 @@
 import org.hisp.dhis.audit.AuditAttributes;
```

Comment:
I did a similar clean-up in this PR: https://github.com/dhis2/dhis2-core/pull/5622

I'll change my PR so you don't have conflicts